clean.cmd

```diff
@@ -0,0 +1 @@
+powershell -ExecutionPolicy Unrestricted scripts\clean.ps1 
```

Comment:
Is there any reason why this script needs to call out to `powershell` to run the git commands? Couldn't it just run them here?

Reply:
You're right! I was following existing examples involving `.cmd` and `.ps1` files. I've moved things over to the `clean.cmd` flle and 🔥 the `clean.ps1`